members/models/dailystatisticsdepartment.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python

```

Comment:
Jeg fjernede **-# -\*- coding: utf-8 -\*-** pga mangel på konveksense.
Og **#!/usr/bin/env python** er et shebang der fortæller et UNIX systemer at når filen eksekveres skal det gøre ved at køre den specificerede kommando med filen som argument, og alle de filer hvor jeg har fjernet shebanget har været biblioteker der ikke skal eksekveres for sig selv.